Yoast/wordpress-seo
# Change verbiage for SEO configured box on dashboard

## Summary

This PR can be summarized in the following changelog entry:

* Improve verbiage for SEO configured box on dashboard so it's not interpreted as a warning.

## Relevant technical choices:

* Also updated the function first_time_notification() for consistency in how the notification title is escaped.

## Test instructions

This PR can be tested by following these steps:

* Install and activate Yoast SEO plugin.
* Run configuration wizard. 
* Return to Yoast SEO dashboard. See new title and content verbiage with checkmark icon as follows:

<img width="632" alt="Screen Shot 2019-07-16 at 12 22 39 PM" src="https://user-images.githubusercontent.com/24902269/61311654-cd38ef80-a7c4-11e9-80b0-0037135e96d0.png">

## UI changes
Unable to add label as a community contributor.

## Documentation

## Quality assurance

I updated the "old" integration test for this verbiage.

Fixes #9476
